hamsm_restarting/pull_sample_data.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+DATA_URL="https://zenodo.org/record/6471213/files/sample_data.tar.gz"
+
+echo "Downloading sample data from Zenodo ... "
+wget $DATA_URL
```

Comment:
```suggestion
wget --continue $DATA_URL
```
Just so people resuming the download don't have to start over.